packages/houdini/src/codegen/transforms/schema.ts

```diff
@@ -52,6 +52,14 @@ directive @${config.paginateDirective}(${config.listOrPaginateNameArg}: String,
 """
 directive @${config.listPrependDirective} on FRAGMENT_SPREAD
 
+"""
+	@${
+		config.dedupeDirective
+	} is used to prevent an operation from running twice. If the operation is sent
+	while there is already one pending and the cancelFirst arg is true, the first one will be canceled instead of the second.
```

Comment:
Maybe something along the lines of this is a little bit clearer?

```suggestion
	@${
		config.dedupeDirective
	} is used to prevent an operation from running more than once at the same time.
	If the cancelFirst arg is set to true, the response already in flight will be canceled instead of the second one.
```

Reply:
Yea much better!